examples/inference_triton/inference-HugeCTR/README.md

```diff
@@ -0,0 +1,113 @@
+# Inference with HugeCTR Model
+
+In this folder, we provide two example notebooks, [movielens-HugeCTR](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-HugeCR.ipynb) and [movielens-HugeCTR-inference](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-inference.ipynb), and explain the steps to do inference with Merlin Inference API after training a model with HugeCTR framework. 
+
+## Getting Started 
+
+In order to use Merlin Inference API, there are two containers that the user needs to build and launch. The first one is for preprocessing with NVTabular and training a model with HugeCTR framework. The other one is for serving/inference. 
+
+## 1. Pulling the NVTabular Docker Container:
+
+We start with pulling NVTabular container. This is to do preprocessing, feature engineering on our datasets using NVTabular, and then to train a DL model with HugeCTR framework with processed datasets.
+
+Before starting docker continer, first create a `nvt_triton` directory and `data` subdirectory on your host machine:
+
+```
+mkdir -p nvt_triton/data/
+cd nvt_triton
+```
+We will mount `nvt_triton` directory into the NVTabular docker container.
+
+Merlin containers are available in the NVIDIA container repository at the following location: http://ngc.nvidia.com/catalog/containers/nvidia:nvtabular.
+
+You can pull the `Merlin-Training` container by running the following command:
+
+```
+docker run --gpus=all -it -v ${PWD}:/model/ -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host nvcr.io/nvstaging/merlin/merlin-training:0.4 /bin/bash
+```
+The container will open a shell when the run command execution is completed. You'll have to start the jupyter lab on the Docker container. It should look similar to this:
+
+```
+root@2efa5b50b909:
+```
+
+Activate the rapids conda environment by running the following command:
+```
+root@2efa5b50b909: source activate rapids
+```
+You should receive the following response, indicating that the environment has been activated:
+
+```
+(rapids)root@2efa5b50b909:
+```
+1) Install Triton Python Client Library:
```

Comment:
Isn't the triton python client library installed on the containers? Can we delete this section